src/pages/consequences.module.scss

```diff
@@ -0,0 +1,45 @@
+@import '../assets/shared-styles/colors.module.scss';
+
+.riskBuying {
+    display: inline-block;
+    font-family: cera-light;
+    font-size:1.8rem;
+    margin: 0.1em 1em 0.1em 1em;
+}
+
+.textWrapper p {
```

Comment:
Do you need the text wrapper div around everything? How about just targeting the `p` tag? It's better not to add in extra divs that don't add structure